src/OneDrive.js

```diff
@@ -269,6 +269,19 @@ class OneDrive extends EventEmitter {
     }
   }
 
+  /**
+   */
+  async getRootFolderId(driveId) {
```

Comment:
this returns a drive item, right? so why is it called `getRootFolderId` ?

better: `getDriveRootItem` ?

Reply:
Right, I changed the method signature without changing its name, will do!